### ISSUE TITLE
Modernize the UI

### ISSUE THREAD

**Issue by MordechaiHadad** [MrAlex94/Waterfox#2286]
**Is your feature request related to a problem? Please describe.**
The current ui is like the older firefox ui which looks pretty dated

**Describe the solution you'd like**
Modernize it and make it a bit more like firefox/chrome

**Comment by q64ceo** [comment]
If they were to do this, I would want an option to go back to the old way of doing things. I'm not a fan of Proton.

**Comment by MordechaiHadad** [comment]
> 
> 
> If they were to do this, I would want an option to go back to the old way of doing things. I'm not a fan of Proton.

fair

**Comment by Felipefpl** [comment]
also, australis is the GREATEST theme firefox ever had. ;)

**Comment by hawkeye116477** [comment]
So you want modern theme in Classic? That won't happen rather. Official statement is that there will be only security updates and the other features may be provided by external contributors, but there aren't many external contributors who want to add something to Classic, mainly only I'm here and I'm not planning to make modern theme.

**Comment by Sollace** [comment]
> 
> 
> If they were to do this, I would want an option to go back to the old way of doing things. I'm not a fan of Proton.

Same here.

Currently G4 Waterfox looks and feels exactly like FF91/Proton, which makes sense, given it's still a WIP and is based on FF91. I have no problem with it being there for the few who like proton, however I would MASSIVELY prefer to have it look and behave like current FF78/Photon (Waterfox G3). That includes the menus, tabs, context menus, settings menus, everything.

I don't want to see anything from Proton. Especially not the new checkboxes in FF91's settings that make it impossible to tell whether they're checked or not.

**Comment by MordechaiHadad** [comment]
> 
> 
> > If they were to do this, I would want an option to go back to the old way of doing things. I'm not a fan of Proton.
> 
> Same here.
> 
> Currently G4 Waterfox looks and feels exactly like FF91/Proton, which makes sense, given it's still a WIP and is based on FF91. I have no problem with it being there for the few who like proton, however I would MASSIVELY prefer to have it look and behave like current FF78/Photon (Waterfox G3). That includes the menus, tabs, context menus, settings menus, everything.
> 
> I don't want to see anything from Proton. Especially not the new checkboxes in FF91's settings that make it impossible to tell whether they're checked or not.

how can I try G4? do i need to build from source?

**Comment by Peacock365** [comment]
@Sollace 

Try the "Lepton" theme under about:addons, it is reminiscent of Photon (the design before Proton).

**Comment by adamp01** [comment]
@MordechaiHadad 

You can find pre-built binaries of the G4 beta versions here: https://cdn.waterfox.net/staging/x86_64/

**Comment by MordechaiHadad** [comment]
> 
> 
> @MordechaiHadad
> 
> You can find pre-built binaries of the G4 beta versions here: https://cdn.waterfox.net/staging/x86_64/

Damn looks exactly how i wanted, should i close the issue?

**Comment by Sollace** [comment]
@Peacock365 
> 
> 
> @Sollace
> 
> Try the "Lepton" theme under about:addons, it is reminiscent of Photon (the design before Proton).

Lepton was already selected when I installed it, but it just seemed to be Proton with changed tab shapes and added icons to the menu. That's not really what I wanted. It also doesn't have my OS colour scheme in the tab bar, something I've only been able to get to work in regular FF by setting `browser.proton.enabled` to false in about:config. (that doesn't disable proton, but does seem to remove the hardcoded off-white background in the tab bar, for some reason)

Also I made the mistake of switching themes in G4, and Lepton got permanently borked back to Proton. so... Uh. Don't do that, lol.
